hs_app_timeseries/forms.py

```diff
@@ -143,38 +245,104 @@ class ProcessingLevelValidationForm(forms.Form):
 
 
 class TimeSeriesResultFormHelper(BaseFormHelper):
-    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None, *args, **kwargs):
-        # the order in which the model fields are listed for the FieldSet is the order these fields will be displayed
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None,
+                 *args, **kwargs):
+        # the order in which the model fields are listed for the FieldSet is the order these
+        # fields will be displayed
         field_width = 'form-control input-sm'
         layout = Layout(
-                     Field('units_type', css_class=field_width),
+                     Field('units_type', css_class=field_width,
+                           title="Select 'Other...' to specify a new units type term"),
                      Field('units_name', css_class=field_width),
                      Field('units_abbreviation', css_class=field_width),
-                     Field('status', css_class=field_width),
-                     Field('sample_medium', css_class=field_width),
+                     Field('status', css_class=field_width,
+                           title="Select 'Other...' to specify a new status term"),
+                     Field('sample_medium', css_class=field_width,
+                           title="Select 'Other...' to specify a new sample medium term"),
                      Field('value_count', css_class=field_width),
-                     Field('aggregation_statistics', css_class=field_width),
+                     Field('aggregation_statistics', css_class=field_width,
+                           title="Select 'Other...' to specify a new aggregation statistics term"),
                      )
         kwargs['element_name_label'] = 'Time Series Result'
-        super(TimeSeriesResultFormHelper, self).__init__(allow_edit, res_short_id, element_id, element_name, layout,  *args, **kwargs)
+        super(TimeSeriesResultFormHelper, self).__init__(allow_edit, res_short_id, element_id,
+                                                         element_name, layout, *args, **kwargs)
 
 
 class TimeSeriesResultForm(ModelForm):
     def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, **kwargs):
+        self.cv_sample_mediums = list(kwargs['cv_sample_mediums'])
+        self.cv_units_types = list(kwargs['cv_units_types'])
+        self.cv_aggregation_statistics = list(kwargs['cv_aggregation_statistics'])
+        self.cv_statuses = list(kwargs['cv_statuses'])
+        kwargs.pop('cv_sample_mediums')
+        kwargs.pop('cv_units_types')
+        kwargs.pop('cv_aggregation_statistics')
+        kwargs.pop('cv_statuses')
```

Comment:
@pkdash Same comments as above regarding combining assignment and pop into one statement.